x-pack/platform/test/plugin_api_integration/test_suites/task_manager/task_management.ts

```diff
@@ -1129,56 +1129,29 @@ export default function ({ getService }: FtrProviderContext) {
         expect(task.runAt).to.eql(scheduledRunAt);
       });
     });
-  });
 
-  // TODO: Add this back in with https://github.com/elastic/kibana/issues/106139
```

Comment:
this issue related to ephemeral tasks which we removed and the issue was already closed.